packages/wonder-blocks-form/components/checkbox.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+// ☑️ A nicely styled checkbox for all your checkbox-y needs
+
+import * as React from "react";
+
+import {getClickableBehavior} from "@khanacademy/wonder-blocks-core";
+import CheckboxCore from "./checkbox-core.js";
+
+import type {ChoiceProps} from "../util/types.js";
+
+export default class Checkbox extends React.Component<ChoiceProps> {
+    static defaultProps = {
+        disabled: false,
+        error: false,
+    };
+
+    render() {
+        const {onChange, ...coreProps} = this.props;
+        const ClickableBehavior = getClickableBehavior(this.context.router);
+
+        return (
+            <ClickableBehavior
+                disabled={coreProps.disabled}
+                onClick={() => onChange(coreProps.checked)}
```

Comment:
I think this should be `onChange(!coreProps.checked)`.  One of the examples should probably show this in action so that we have something to test even if we're going to use `CheckboxGroup` in the future.

Reply:
I agree, this is actually changed to `!coreProps.checked` in a later diff.

Reply:
I'll add a test in the #236, which would use the new signature.